test/python/quantum_info/test_pauli_lindblad_map.py

```diff
@@ -1231,6 +1231,139 @@ def test_signed_sample(self):
         self.assertEqual(len(qubit_sparse_pauli_list), 5)
         self.assertEqual(len(signs), 5)
 
+    def test_parity_sample(self):
+
+        # test all negative rates
+        pauli_lindblad_map = PauliLindbladMap([("X", -1.0), ("Y", -1.0)])
+        probs = pauli_lindblad_map.probabilities()
+        probs_dict = {
+            "I": probs[0] * probs[1],
+            "X": probs[0] * (1 - probs[1]),
+            "Y": (1 - probs[0]) * probs[1],
+            "Z": (1 - probs[0]) * (1 - probs[1]),
+        }
```

Comment:
In this test (and many tests in this file), are the probabilities for `X` and `Y` errors accidentally swapped? Should it be like this instead (since it looks like the chosen convention is to work with probabilities of each error NOT happening)?

```suggestion
        pauli_lindblad_map = PauliLindbladMap([("X", -1.0), ("Y", -1.0)])
        probs = pauli_lindblad_map.probabilities()
        probs_dict = {
            "I": probs[0] * probs[1],
            "X": (1-probs[0]) * probs[1],
            "Y": probs[0] * (1-probs[1]),
            "Z": (1 - probs[0]) * (1 - probs[1]),
        }
```

Reply:
Great catch on this. This was initially very concerning but it turns out I wrote every single test case with `probs[0] == probs[1]`, so my incorrect formulas are still producing the right values.

I'll implement this fix and also change the values a bit so that this would have failed with the incorrect formulas.

Reply:
For the Python test code around lines 1240–1250, may be you can consider adding a clear docstring specifying the intended error probability convention, parameterizing the test to check both canonical and swapped orderings, and expanding assertion messages for clarity. This can catch subtle bugs and ensure test robustness. Additionally, helper functions for probability indexing would help maintain consistency across the test suite, i guess.